enhancements/baremetal/an-slo-for-baremetal.md

```diff
@@ -315,6 +326,7 @@ We need the following changes to MAO:
 
    In `4.N-dev`, once the CBO has been added to the release, we can
    remove all awareness of the `metal3` deployment from MAO.
+   Note that this will not be removed in version 4.7.
```

Comment:
We do plan to remove all awareness of metal3 from MAO in 4.7.

Reply:
Ok